OpenGLViewSample/OpenGLViewSample/AppDelegate.cs

```diff
@@ -18,16 +18,17 @@ public AppDelegate ()
 		{
 		}
 
-		public override void DidFinishLaunching (NSNotification notification)
+		public override void DidFinishLaunching(NSNotification notification)
 		{
-			mainWindowController = new MainWindowController ();
-			view = new GLView (mainWindowController.Window.Frame, new NSOpenGLPixelFormat (new object [] {
+			mainWindowController = new MainWindowController();
+			Class.ThrowOnInitFailure = false;
```

Comment:
This is not something we should be showing in a sample, not without at least a large comment block explaining why we need it.